src/OmniSharp/Workers/Intellisense/SnippetGenerator.cs

```diff
@@ -75,9 +78,9 @@ private void RenderMethodSymbol(IMethodSymbol methodSymbol)
                 }
                 _sb.Append(">");
             }
-            
+
             RenderParameters(methodSymbol);
-            if (methodSymbol.ReturnsVoid)
+            if (methodSymbol.ReturnsVoid && IncludeMarkers)
```

Comment:
I don't like this. I want to see the ; in the display text as it will be inserted.